src/api/.keystone/config.js

```diff
@@ -170,7 +170,14 @@ var lists = {
           { label: APPLICATION.POLICY_TYPE.MULTI, value: APPLICATION.POLICY_TYPE.MULTI }
         ]
       }),
-      requestedStartDate: (0, import_fields.timestamp)()
+      requestedStartDate: (0, import_fields.timestamp)(),
+      totalValueOfContract: (0, import_fields.integer)({
+        validation: {
+          min: 1,
+          max: 499999
```

Comment:
Is this `max`  value correct? Does this mean we can not have a total value of contract over £499,999?

Reply:
Yes that's right - business requirement.

I will move this to constants actually

Reply:
Moved to constants - marking as resolved